Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -308,7 +308,11 @@ struct VariadicsGenerator: ParsableCommand {
     output("""
         {
           let factory = makeFactory()
-          return factory.accumulate(accumulated, next)
+          if #available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *) {
+            return factory.accumulate(accumulated, factory.ignoreCapturesInTypedOutput(next))
+          } else {
+            return factory.accumulate(accumulated, next)
+          }
```

Comment:
How about a helper function that will reduce spew and, more importantly, give a name and place to commend why we're doing this and how it differs across versions.

E.g.

```swift
// comment...
private func dropCaptures(_ next: ...) -> ... {
  // ... comment about old and new behavior
  if #available(...) {
   return  ...
  }
  return ...
}

...

return factory.accumulate(accumulated, dropCaptures(next))
```

(or some better name than `dropCaptures`)

Reply:
That's a good call, I only ever wrote this once, but it is codegen'd an awful lot.